### DISCUSSION TITLE
[OUTDATED] Abstract AtlasOS Wallpapers

### DISCUSSION THREAD

**Discussion by meetox80** [Atlas-OS/Atlas#627] [Community Artwork]
# AtlasOS Wallpaper HUB
- lemonekq/atlasos is EOL, moved right there. Feel free to copy, edit and modify these wallpapers into your own use.
- new wallpapers would be added and updated soon. **give me ideas for them!**

# Abstract

- Explode

> Dark Purple
![explode_dark_purple](https://user-images.githubusercontent.com/72318911/228944132-a2acb964-5108-40c2-83d7-5f99d38ed5d7.png)
[explode_dark_purple.zip](https://github.com/Atlas-OS/Atlas/files/11115385/explode_dark_purple.zip)

> Dark Red
![explode_dark_red](https://user-images.githubusercontent.com/72318911/228945151-09c8d1fa-d74d-4351-a8ff-25073a029b95.png)
[explode_dark_red.zip](https://github.com/Atlas-OS/Atlas/files/11115477/explode_dark_red.zip)

> White Purple
![explode_white_purple](https://user-images.githubusercontent.com/72318911/228945634-fddc9c6a-e5aa-4aad-b522-ec2dbdfd6ab7.png)
[explode_white_purple.zip](https://github.com/Atlas-OS/Atlas/files/11115444/explode_white_purple.zip)

> White Red
![explode_white_red](https://user-images.githubusercontent.com/72318911/228945912-010cf2d2-a0f8-4203-9c51-da1315773ee1.png)
[explode_white_red.zip](https://github.com/Atlas-OS/Atlas/files/11115448/explode_white_red.zip)

# Patterns

- Repeat

> Repeat Dark
![repeat_dark](https://user-images.githubusercontent.com/72318911/228946406-8dae4de1-578c-4d35-8600-ce9944af6fdf.png)
[repeat_dark.zip](https://github.com/Atlas-OS/Atlas/files/11116172/repeat_dark.zip)

> Repeat White
![repeat_white](https://user-images.githubusercontent.com/72318911/228966926-92b3247d-85b2-4dd4-bfa2-642ba493ccf7.png)
[repeat_white.zip](https://github.com/Atlas-OS/Atlas/files/11116188/repeat_white.zip)

- Rise
> Dark Blue
![rise_dark_blue](https://user-images.githubusercontent.com/72318911/228954183-2b2383c4-6367-4aa9-ace1-c28405d037b7.png)
> No psd/zip embedded.

> Matte Blue
![rise_matte_blue](https://user-images.githubusercontent.com/72318911/228953959-5509fab7-e9ab-43e4-a3ff-931ba0ebb3ce.png)
> No psd/zip embedded.

> White Blue
![rise_white_blue](https://user-images.githubusercontent.com/72318911/228954024-1ddaf9c4-dc46-4168-83e3-a3e791567f9a.png)
> No psd/zip embedded.

> Dark Red
![rise_dark_red](https://user-images.githubusercontent.com/72318911/228954229-8c3bb6a5-e012-4177-9214-53fb0c9a4049.png)
> No psd/zip embedded.

> Matte Red
![rise_matte_red](https://user-images.githubusercontent.com/72318911/228954292-be1e8aea-0fbf-4b75-94df-6ee566808d66.png)
> No psd/zip embedded.

> White Red
![rise_white_red](https://user-images.githubusercontent.com/72318911/228954383-cfbcedec-9012-4447-bfd4-31c1442e0500.png)
> No psd/zip embedded.

# Classics

- Valo

> Valo Dark
![valo_dark](https://user-images.githubusercontent.com/72318911/228946635-e8f22eaf-ebb8-4a32-8940-c2a7e79efa93.png)
[valo_dark.zip](https://github.com/Atlas-OS/Atlas/files/11115469/valo_dark.zip)

> Valo White
![valo_white](https://user-images.githubusercontent.com/72318911/228946711-2a30874b-829d-4103-b976-1449353f526c.png)
[valo_white.zip](https://github.com/Atlas-OS/Atlas/files/11115474/valo_white.zip)

- Bubble

> Bubble Dark Purple
![bubble_dark_purple](https://user-images.githubusercontent.com/72318911/228972445-6b86f60c-fb00-4cf7-8674-085f4ca6147e.png)
[bubble_dark_purple.zip](https://github.com/Atlas-OS/Atlas/files/11116446/bubble_dark_purple.zip)

> Bubble Matte Purple
![bubble_matte_purple](https://user-images.githubusercontent.com/72318911/228973649-f6f7c5d1-8b5f-4827-b9bc-1625180b656a.png)
[bubble_matte_purple.zip](https://github.com/Atlas-OS/Atlas/files/11116491/bubble_matte_purple.zip)

> Bubble White Purple
![bubble_white_purple](https://user-images.githubusercontent.com/72318911/228972626-8dd77c97-157c-423d-a460-61e5c6c24cd0.png)
[bubble_white_purple.zip](https://github.com/Atlas-OS/Atlas/files/11116456/bubble_white_purple.zip)

> Bubble Dark Red
![bubble_dark_red](https://user-images.githubusercontent.com/72318911/228972782-6aa81253-5957-4167-9996-ec80036165ed.png)
[bubble_dark_red.zip](https://github.com/Atlas-OS/Atlas/files/11116458/bubble_dark_red.zip)

> Bubble Matte Red
![bubble_matte_red](https://user-images.githubusercontent.com/72318911/228973923-ac2b59d0-e3f1-4236-843f-bf80338f9f75.png)
[bubble_matte_red.zip](https://github.com/Atlas-OS/Atlas/files/11116496/bubble_matte_red.zip)

> Bubble White Red
![bubble_white_red](https://user-images.githubusercontent.com/72318911/228972857-1084ccd1-2ac5-4b55-b1ae-ef5b9c23806b.png)
[bubble_white_red.zip](https://github.com/Atlas-OS/Atlas/files/11116460/bubble_white_red.zip)

**Comment by GoraLaxor44** [comment]
nice Wallpapers!
